packages/@aws-cdk/aws-apigatewayv2/lib/http/api.ts

```diff
@@ -118,7 +127,11 @@ export class HttpApi extends Resource implements IHttpApi {
   }
 
   public readonly httpApiId: string;
-  private readonly defaultStage: HttpStage | undefined;
+  /**
+   * default stage of the api resource
+   */
+  public readonly defaultStage: HttpStage | undefined;
```

Comment:
Why are we changing this to public?

I would prefer to be conservative about what we expose on the publicly API.

Reply:
We probably can create `HttpApiMapping` resource like this?

https://github.com/aws/aws-cdk/blob/38edd9329356902fbf5ddf6ebdf9641cdfedd89d/packages/%40aws-cdk/aws-apigatewayv2/test/http/api-mapping.test.ts#L20-L24